cmd/add.go

```diff
@@ -12,8 +12,8 @@ var addCmd = &cobra.Command{
 	Long: `This command will add a directory to the project database.
     The directory will not be added if it does not contain a README.md.
     `,
-	Run: func(cmd *cobra.Command, args []string) {
-		pkg.InitDirs(args)
+	RunE: func(_ *cobra.Command, args []string) error {
```

Comment:
it it go convention to suffix function names with E when they are returning an error or just a good practice in general  ?

Reply:
No. I would say that go convention is to always return an error.

I'm unsure why cobra has Run signature without an error.

Reply:
I mean the cobra code is very complex because of that

https://github.com/spf13/cobra/blob/5a1acea3210649f3d70002818ec04b09f6347062/command.go#L969-L975